pt-BR/mail/chrome/messenger/addressbook/abSelectAddressesDialog.dtd

```diff
@@ -36,32 +36,31 @@
  ***** END LICENSE BLOCK ***** -->
 
 <!-- Title -->
-<!ENTITY selectAddressWindow.title           "Select Addresses">
+<!ENTITY selectAddressWindow.title           "Selecione os endereços">
 
 <!-- Buttons -->
-<!ENTITY toButton.label                      "To->">
+<!ENTITY toButton.label                      "Para->">
 <!ENTITY toButton.accesskey                  "T">
 <!ENTITY ccButton.label                      "Cc->">
 <!ENTITY ccButton.accesskey                  "C">
-<!ENTITY bccButton.label                     "Bcc->">
+<!ENTITY bccButton.label                     "Cco->">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY bccButton.label                     "Bcc->">
@brunotiussi please don't translate the string Bcc.